project/os.scala

```diff
@@ -126,7 +126,7 @@ object git {
   }
 
   def distance(from: String, to: String): Int = {
-    shell.check_output(s"git rev-list $to ^$from --count", cwd = ".").trim.toInt
+    shell.check_output(s"git rev-list $to $from --count", cwd = ".").trim.toInt
```

Comment:
Because the command kept crashing. I don't know why this behavior changed, it could be caused by the git version installed on the docker image.